### DISCUSSION TITLE
specification should include a format version in each file

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#48] [Q&A] [answered]
Since the data format of the files may be updated, it would be beneficial to include a version number or version identifier in each file that would identify the version of the format that the specific file conforms to.

**Comment by shaselton-usds** [comment]
Good suggestion and definitely been in the back of our mind. Would you envision this being nothing more than a `version` attribute on the base object?

**Reply by tdfow** [reply]
I would concur with a version attribute on the base object to denote the version number/identifier.   Would the publication of the schema then identify/reflect the version number/identifier?

**Reply by nmayle** [reply]
Major and minor version numbers seem typical of specs.  Documenting the different versions of the schema seems like a good idea.

**Comment by shaselton-usds** [comment, marked as answer]
A version was included! Thanks for the feedback. You can find the change here: https://github.com/CMSgov/price-transparency-guide/commit/18179661b4e96e7034477bdabe60e63522c7fad1#diff-5788d606f767177de0b701c945124d35072c320957a18a6d7a3db2e52f11e442
